raspberry/bzzz/sensors/evo_time_of_flight.py

```diff
@@ -77,11 +77,11 @@ def __get_measurements_in_background_t(self, serial_path, baud):
 
                 # Checking error codes
                 if rng == 65535:  # Sensor measuring above its maximum limit
-                    dec_out = float('inf')
+                    dec_out = -1
```

Comment:
We've changed these to `nan`, right?

Reply:
@jamie-54 bump

Reply:
this issue is fixed in the refactoring so I'm going to close this issue